Admin/Field/ArrayField.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace LAG\AdminBundle\Admin\Field;
+
+use LAG\AdminBundle\Admin\Field;
+use Exception;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Traversable;
+
+/**
+ * Array field.
+ *
+ * Note : class can not be call Array by php restriction
+ */
+class ArrayField extends Field
```

Comment:
There might be something to think differently on these fields, could we any features of the form API?